web/src/billing/billing.ts

```diff
@@ -1,5 +1,7 @@
 import $ from "jquery";
 
+import * as overlays from "../overlays";
```

Comment:
I don't want to import `overlays.ts` in portico code; it has a lot of code that isn't designed to be used outside the app. I think a thing we could maybe plausibly do is split `overlays.ts` so that there's a separate `modals.ts` that is just the modals library -- that's probably an overdue split regardless.

But I think my preference would be to do something more isolated just using the upstream APIs, like we have in `web/src/portico/signup.ts`, if that doesn't require too much extra code, just because that isolation protects us from accidentally breaking the billing pages (which we aren't going to manually test often).